frontend/src/old-pages/Clusters/Clusters.tsx

```diff
@@ -27,44 +25,53 @@ import {
   Table,
   TextFilter
 } from "@awsui/components-react";
-
 import { useCollection } from '@awsui/collection-hooks';
 
-// Components
 import EmptyState from '../../components/EmptyState';
 import Status from "../../components/Status";
 import Actions from './Actions';
 import Details from "./Details";
 import { wizardShow } from '../Configure/Configure';
 
-function updateClusterList() {
+
+export interface Cluster {
```

Comment:
suggestion: it is good to extract types into separate files and import them when needed - this has the advantage of avoiding circular imports in some edge cases